tailwind.config.js

```diff
@@ -28,7 +28,8 @@ module.exports = {
       gray: "#6c6868",
       lightGray: "#c4c2c2",
       error: "#ff0000",
-      neutral: "#E5E5E5"
+      neutral: "#E5E5E5",
+      red: "#ff0000"
```

Comment:
There is already a color here with the same hex code. So you can reference the color `error` instead of red